app/scripts/controllers/transactions/lib/recipient-blocklist-checker.js

```diff
@@ -7,8 +8,7 @@ import blocklist from './recipient-blocklist'
  * @throws {Error} if the account is blocked on mainnet
  */
 export function throwIfAccountIsBlocked (networkId, account) {
-  const mainnetId = 1
-  if (networkId !== mainnetId) {
+  if (networkId !== MAINNET_NETWORK_ID) {
```

Comment:
Note that this check was always `true` before this change, as it incorrectly compared the number `1` to the `networkId`, which is a string.

But this remains broken for various other reasons, so 🤷‍♂️ It's not super important to me that this be in a separate PR. I'll have one up shortly after this to remove this module altogether, since it doesn't work and isn't worth the effort to fix right now.

Reply:
This module is removed in #8943